libs/core/langchain_core/callbacks/stdout.py

```diff
@@ -32,8 +32,14 @@ def on_chain_start(
             inputs (Dict[str, Any]): The inputs to the chain.
             **kwargs (Any): Additional keyword arguments.
         """
-        class_name = serialized.get("name", serialized.get("id", ["<unknown>"])[-1])
-        print(f"\n\n\033[1m> Entering new {class_name} chain...\033[0m")  # noqa: T201
+        if "name" in kwargs:
```

Comment:
We can fetch the `name`  from kwargs here if present and otherwise fallback to serialized representation